lib/CL/devices/CMakeLists.txt

```diff
@@ -39,6 +39,12 @@ if(BUILD_PTHREAD)
   add_subdirectory("pthread")
 endif()
 
+if(BUILD_TBB)
+  include_directories(AFTER "${TBB_INCLUDE_DIRS}")
+  add_subdirectory("tbb")
+  list(APPEND POCL_DEVICES_LINK_LIST ${TBB_LIBRARIES})
```

Comment:
why does  this differ from the other devices (l 43, 45)?

Reply:
L43 is necessary because the `tbb` device uses the TBB library. L45 is not needed and has been discarded. Let me know if there is a more elegant way to handle libraries.

Reply:
The build system has been updated and the integration of the TBB library looks much more elegant now.